packages/avatars/demo/~patterns/stories/MenuStory.tsx

```diff
@@ -64,7 +64,9 @@ export const MenuStory: StoryFn = ({ isCompact }) => {
                     status={item.avatarProps.status}
                     badge={item.avatarProps.badge}
                     surfaceColor={
-                      highlightedValue === item.value ? 'background.primary' : 'background.raised'
+                      highlightedValue === item.value
+                        ? 'background.primaryEmphasis'
```

Comment:
I'm waiting for the PR to deploy, but I don't think this is what we want to demonstrate with this pattern. The idea is to get the avatar's surface to match the active menu item as close as possible. So, it should be something like `getColor({ theme, hue: 'primaryHue', light: { shade: 100}, dark: { shade: 900} })` to correspond with the previous variable values.

Reply:
Or maybe `getColor({ theme, variable: 'background.primaryEmphasis', transparency: 100 })` is a more successful match 🤷

Reply:
> Or maybe getColor({ theme, variable: 'background.primaryEmphasis', transparency: 100 }) is a more successful match 🤷

`getColor` will throw because the output of the ☝️  is `rgba()` but it treats it as a color variable (`$surfaceColor.includes('.')`). We would have to refine the logic everywhere this comes up.

```
ReferenceError: Error: color variable 'rgba(31,115,183,0.08)' is not defined
```

Also, with the way `Avatar` is built, the transparent `surfaceColor` would lay on top of the background color. That wouldn't result in the right color.